sound/soc/codecs/rt715-sdca-sdw.c

```diff
@@ -0,0 +1,301 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
WARNING: Improper SPDX comment style for 'sound/soc/codecs/rt715-sdca-sdw.c', please use '//' instead